digital-deck-ui/src/components/PlayerHand.vue

```diff
@@ -28,6 +29,13 @@ export default {
         ? CardDeckImageEnum.FRONT[card - 1]
         : CardDeckImageEnum.BACK.ONE;
```

Comment:
I didn't notice that we already supported card flip. I know this isn't part of your commit, but can you please add a `DEFAULT` that links to `https://i.imgur.com/RO3DOEa.jpg` under `CardDeckImageEnum.BACK`, and replace it here and in `TableArea.vue`? The image right now looks weird because it was meant for the deck, not individual cards.